apps/shopify/src/basePagination.js

```diff
@@ -34,7 +34,9 @@ export default class BasePagination {
       pagination: {
         hasNextPage: this.hasNextProductPage,
       },
-      products: products.map(this.dataTransformer),
+      products: products.map((product) => {
+        return this.dataTransformer(product);
+      }),
```

Comment:
this was originally causing the dataTransformer's `apiEndpoint` to be populated with the index of the current product